### DISCUSSION TITLE
Documentation use of "#### Example:" and "#### Examples:"

### DISCUSSION THREAD

**Discussion by hasezoey** [Automattic/mongoose#12316] [General]
Currently many jsdoc comments use either `Examples:` or `Example:`, not regarding how many examples there are, should this maybe be changed to be consistent to just `Example:` (or something else)?

Some Examples:
https://github.com/Automattic/mongoose/blob/2de1a86bf90c8d97ebfcde12e2fc465a85c489c0/lib/aggregate.js#L243-L250
https://github.com/Automattic/mongoose/blob/2de1a86bf90c8d97ebfcde12e2fc465a85c489c0/lib/aggregate.js#L156-L174
https://github.com/Automattic/mongoose/blob/2de1a86bf90c8d97ebfcde12e2fc465a85c489c0/lib/aggregate.js#L89-L101
https://github.com/Automattic/mongoose/blob/2de1a86bf90c8d97ebfcde12e2fc465a85c489c0/lib/aggregate.js#L20-L45

PS: this is a discussion because i would like to change it to be consistent, but would like to know what to change it to before doing it

**Comment by vkarpov15** [comment]
I think switching to consistent `Example: ` is fine.

**Reply by hasezoey** [reply]
created PR #12432
